.circleci/config.yml

```diff
@@ -54,9 +54,70 @@ orbs:
   slack: circleci/slack@4.10.1
   shellcheck: circleci/shellcheck@3.2.0
   codecov: codecov/codecov@5.0.3
-  utils: ethereum-optimism/circleci-utils@1.0.8
```

Comment:
Should we fork `ethereum-optimism/circleci-utils` and change in the forked repo instead?

Reply:
The benefit of using an orb is the reuse of predefined commands, especially across organizations. 

To fork the code, we would need to maintain an additional repository and configure settings in the web UI. 

I think it's more convenient to use it as a command for simplicity, especially because we only utilize a small part of the entire circleci-utils orb.